recipes/pdal/all/conanfile.py

```diff
@@ -0,0 +1,122 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+class PdalConan(ConanFile):
+    name = "pdal"
+    description = "PDAL is Point Data Abstraction Library. GDAL for point cloud data."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://pdal.io"
+    license = "BSD"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "with_unwind": [True, False],
+               "with_xml": [True, False],
+               "with_zstd": [True, False],
+               "with_laszip": [True, False],
+               }
+    default_options = {"shared": False,
```

Comment:
```suggestion
    default_options = {"shared": True,
```

Static is part of the default package id. If it doesn't work, it will be better use shared, otherwise any other package which use PDAL will need to force its option.

Reply:
Ok, it will be easier to use. Thought 'shared=True' as default is not encouraged.

Reply:
Ah, hook complains

Reply:
🤯

Reply:
It's interesting how the `tbb` recipe is built with such default option.

Reply:
You need to open an issue against the hooks repo to add an exception as i recall

Reply:
Yes, already opened

Reply:
Hook is working